lib/_http_server.js

```diff
@@ -331,6 +331,11 @@ ServerResponse.prototype._implicitHeader = function _implicitHeader() {
 
 ServerResponse.prototype.writeHead = writeHead;
 function writeHead(statusCode, reason, obj) {
+
+  if (this._header) {
+    throw new ERR_HTTP_HEADERS_SENT('writeHead');
```

Comment:
The PR looks fine.
Can you just rework the sentence here so that the final error message makes full sense in English (see tests)?

Pre-approving tho.